src/Header.js

```diff
@@ -7,23 +7,6 @@ import SearchByParlor from './SearchByParlor';
 class Header extends Component {
   constructor() {
     super();
```

Comment:
@hlhartley was removing all of this code intentional?

Reply:
I think I figured it out.
In one of the previous pull requests the merge conflicts were unresolved before merging to master. That's why we were having duplicates.
I resolved the conflicts in this separate branch, deleting the duplicates, which is why it shows that code being deleted. However, the code we still want to keep (ex. in Header.js is still there - you can check the branch file and go into Header.js before merging this).
If you merge this branch, we should have no more conflicts.

Reply:
Sounds good! I will merge and we can clean up anything unexpected that we overlooked.